Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.py

```diff
@@ -77,6 +77,10 @@ def responders_to_json(responders: List, responder_key: str, one_is_dict: bool =
 
     def create_alert(self, args: dict):
         args['responders'] = argToList(args.get('responders'))
+        if not isinstance(args.get('details'), dict):
+            args['details'] = {key_value.split('=')[0]:key_value.split('=')[1]
+                               for key_value in args.get('details').split(',')}
```

Comment:
```suggestion
        if args.get('details') and not isinstance(args.get('details'), dict):
            args['details'] = {key_value.split('=')[0]: key_value.split('=')[1]
                               for key_value in argToList(args.get('details'))}
```
There should be something like that, `details` is not a mandatory argument